apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -52,6 +61,17 @@ const App: React.FC = () => {
 			<Loader isLoading={dataStatus === DataStatus.PENDING} withOverlay />
 
 			<img alt="logo" className="App-logo" src={reactLogo} width="30" />
+			{isRoot && (
+				<div className={styles["start-meeting"]}>
+					<div>
+						<Button label="Start a meeting" onClick={handleOpenModal} />
+					</div>
```

Comment:
Don we need an extra nesting here?
```suggestion
			    <Button label="Start a meeting" onClick={handleOpenModal} />
```